docs/content/PageLayout.mdx

```diff
@@ -367,7 +367,12 @@ See [storybook](https://primer.style/react/storybook?path=/story/layout-pagelayo
     defaultValue="false"
     description="Whether the pane should stick to the top of the screen while the content scrolls."
   />
-  <PropsTableRow name="stickyTop" type="number" defaultValue="0" description="The height of the sticky top element" />
+  <PropsTableRow
+    name="stickyTop"
+    type="number | string"
+    defaultValue="0"
+    description="Use stickyTop to push the sticky pane down to make room for a sticky header if necessary"
```

Comment:
Should we add into the description that if it is number, we expect px and if it is string value with units?

Reply:
Yeah, good idea!

Reply:
Done! https://github.com/primer/react/pull/2232/commits/800b3eca892a51c5803715ef99dee296946eb3d6 let me know if you have a better wording :)